test/CompactStringVectorTest.cpp

```diff
@@ -109,27 +109,27 @@ TEST(CompactVectorOfStrings, Serialization) {
 TEST(CompactVectorOfStrings, SerializationWithPush) {
   auto testSerializationWithPush = []<typename V>(const V&, auto& inputVector) {
     {
-      typename V::Writer writer{"_writerTest.dat"};
+      typename V::Writer writer{"_writerTest2.dat"};
       for (const auto& s : inputVector) {
         writer.push(s.data(), s.size());
       }
     }  // The constructor finishes writing the file.
 
     V compactVector;
-    ad_utility::serialization::FileReadSerializer ser{"_writerTest.dat"};
+    ad_utility::serialization::FileReadSerializer ser{"_writerTest2.dat"};
     ser >> compactVector;
 
     vectorsEqual(inputVector, compactVector);
 
-    ad_utility::deleteFile("_writerTest.dat");
+    ad_utility::deleteFile("_writerTest3.dat");
```

Comment:
Shouldn't this be `_writerTest2.dat`? There you have your motivation for factoring out the variable.